python/graphstorm/model/rgat_encoder.py

```diff
@@ -118,6 +133,10 @@ def _apply(ntype, h):
                 h = h + self.h_bias
             if self.activation:
                 h = self.activation(h)
+            if self.num_gnn_ngnn_layers != 0:
+                for layer in self.ngnn_gnn:
+                    h = th.matmul(h, layer)
+                h = self.ngnn_activation(h)
```

Comment:
can we implement a separate MLP module?

Reply:
We can have a separate MLP module here. But NGNN supposes to be an option to increase parameters in the GNN. Will it be better to be implemented inside each layer instead of implementing another layer?

Reply:
I would try to implement another module, and let RGCN and RGAT to invoke them separately.

Reply:
you already have the argument `num_gnn_ngnn_layers` that can turn on and off MLP in the GNN module. you can increase two more arguments to turn on and off MLP in the input layer and the decoder.

Reply:
Got it. I can do that in this PR as well.